### DISCUSSION TITLE
StreamedResponse support

### DISCUSSION THREAD

**Discussion by Tryfciu** [dedoc/scramble#146] [Ideas]
Hello,

is there any chance that StreamedResponse/File downloading will be supported?

**Comment by romalytvynenko** [comment]
Hey @Tryfciu! Please share your code examples and actual/expected docs so I can add what you need.

Thanks!

**Reply by Tryfciu** [reply]
I'm basing on this stack overflow thread: https://stackoverflow.com/questions/52538867/model-openapi-2-0-file-download-type-stringbinary-vs-type-file

It states that for file download following schema should be provided:
```
schema:
     type: string
     format: binary
```

Therefore provided schema should be generated for actions that return `StreamedResponse` e.g.

![image](https://github.com/dedoc/scramble/assets/28263270/7cf5f529-d920-4a39-8faa-85e72c91daba)

**Reply by LeTamanoir** [reply]
Hey maybe it could also detect when a controller returns a `\Symfony\Component\HttpFoundation\BinaryFileResponse`

**Comment by arinamirosha** [comment]
also have such problem: in UI response results with 'No supported response body returned'. Where are no promblem with images and txt, but cannot interact with pdf, zip, doc etc.
